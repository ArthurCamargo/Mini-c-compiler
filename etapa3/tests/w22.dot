digraph G {
n0;
n0 -> n1;
n1;
n1 -> n2;
n1 -> n3;
n2;
n2 [label="i"];
n3;
n3 -> n4;
n3 -> n5;
n4;
n4 [label="i"];
n5;
n5 [label="2"];
n3 [label="="];
n1 [label="while"];
n0 [label="f"];

}