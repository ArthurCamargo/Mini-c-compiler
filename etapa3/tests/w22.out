0x5560ea575350
0x5560ea575350, 0x5560ea5755b0
0x5560ea5755b0
0x5560ea5755b0, 0x5560ea575450
0x5560ea5755b0, 0x5560ea575560
0x5560ea575450
0x5560ea575450 [label="i"];
0x5560ea575560
0x5560ea575560, 0x5560ea5754c0
0x5560ea575560, 0x5560ea575510
0x5560ea5754c0
0x5560ea5754c0 [label="i"];
0x5560ea575510
0x5560ea575510 [label="2"];
0x5560ea575560 [label="="];
0x5560ea5755b0 [label="while"];
0x5560ea575350 [label="f"];
